domain/src/main/kotlin/com/patrykandpatryk/liftapp/domain/model/AllPreferences.kt

```diff
@@ -0,0 +1,9 @@
+package com.patrykandpatryk.liftapp.domain.model
+
+import com.patrykandpatryk.liftapp.domain.unit.DistanceUnit
+import com.patrykandpatryk.liftapp.domain.unit.MassUnit
+
+data class AllPreferences(
+    val massUnit: MassUnit,
+    val distanceUnit: DistanceUnit,
+)
```

Comment:
Potential detekt error.
```suggestion
}

```